pyccl/halos/massdef.py

```diff
@@ -225,47 +229,52 @@ def translate_mass(self, cosmo, M, a, m_def_other):
                 D_this = self.get_Delta(cosmo, a) * om_this
                 c_this = self._get_concentration(cosmo, M, a)
                 R_this = self.get_radius(cosmo, M, a)
-                om_new = omega_x(cosmo, a, m_def_other.rho_type)
-                D_new = m_def_other.get_Delta(cosmo, a) * om_new
-                c_new = convert_concentration(cosmo, c_this, D_this, D_new)
+                om_new = omega_x(cosmo, a, mass_def_other.rho_type)
+                D_new = mass_def_other.get_Delta(cosmo, a) * om_new
+                c_new = convert_concentration(cosmo, c_old=c_this,
+                                              Delta_old=D_this,
+                                              Delta_new=D_new)
                 R_new = c_new * R_this / c_this
-                return m_def_other.get_mass(cosmo, R_new, a)
+                return mass_def_other.get_mass(cosmo, R_new, a)
 
 
 class MassDef200m(MassDef):
     """`MassDef` class for the mass definition with Delta=200 times the matter
     density.
 
     Args:
-        c_m (string): concentration-mass relation.
+        c_m_relation (string): concentration-mass relation.
     """
-    def __init__(self, c_m='Duffy08'):
+    @warn_api()
```

Comment:
should you rename `c_m`->`c_m_reation` in `warn_api`?

Reply:
OK